### PR TITLE
Sort mentorlist2

### PR DESCRIPTION
**Pull request by djflexible**

### CODE REVIEW

File: src/server/models/users.model.js [hunk 1]
```diff
@@ -59,7 +59,11 @@ let userSchema = new Schema({
...
   profile_picture: String,
   job_position: String,
-  date: {
+  reg_date: {
+    type: Date,
+    default: Date.now,
+  },
+  stamp_login: {
     type: Date,
     default: Date.now,
```
Comment (lyw0149) [inline src/server/models/users.model.js]: if stamp_login's default is Date.now, stamp will fixed same with reg_date. Delete this line, and stamp login time manually when login

File: src/server/controllers/users.controller.js [hunk 2]
```diff
@@ -118,6 +118,7 @@ function storeSession(req, res, user) {
   req.session.access_token = req.body.access_token;
   req.session.email = user.email;
   req.session._id = user._id.toString();
+  req.session.date = user.stamp_login;
```
Comment (lyw0149) [inline src/server/controllers/users.controller.js]: There is no code for stamp time at login. You should update DB when login using save().exec() or update().exec().

Comment (lyw0149) [inline src/server/controllers/users.controller.js]: And variable name `date` is confusing.

File: src/server/models/users.model.js (hunk 1 above)
Comment (djflexible) [inline src/server/models/users.model.js]: I got it!

File: src/server/controllers/users.controller.js (hunk 2 above)
Comment (djflexible) [inline src/server/controllers/users.controller.js]: I got it!

**Comment by lyw0149** [comment]
LGTM 
We can stamp time when signin!! thank you @djflexible.

**Pull request merged**